fdbserver/workloads/BackupAndParallelRestoreCorrectness.actor.cpp

```diff
@@ -40,6 +41,7 @@ struct BackupAndParallelRestoreCorrectnessWorkload : TestWorkload {
 	bool locked;
 	bool allowPauses;
 	bool shareLogRange;
+	bool usePartitionedLogs;
```

Comment:
I like this! 

Since AtomicRestore.actor.cpp is also used to test the new restore, can you add this to that workload as well?

Reply:
Done.